account_product_move/views/account_product_move.xml

```diff
@@ -0,0 +1,96 @@
+                    </h1>
+                    <group name="config" colspan="4" col="4">
+                        <field
+                            name="product_tmpl_id"
+                            colspan="4"
+                            options="{'no_create_edit': True, 'no_open': True}"
+                        />
+                        <field name="journal_item_ids" colspan="4">
+                            <tree editable="bottom">
+                                <field
+                                    name="account_id"
+                                    options="{'no_create_edit': True, 'no_open': True}"
+                                />
+                                <field name="debit" />
+                                <field name="credit" />
+                                <field name="journal_id" />
+                                <field
+                                    name="currency_id"
+                                    options="{'no_create_edit': True, 'no_open': True}"
+                                />
+                                <field name="product_move_id" invisible="1" />
+                            </tree>
+                        </field>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="account_product_move_tree_view">
+        <field name="name">account.product.move.tree</field>
+        <field name="model">account.product.move</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="product_tmpl_id" string="Product" />
+                <field name="journal_item_ids" />
+            </tree>
+        </field>
+    </record>
+    <!-- Search -->
+    <record model="ir.ui.view" id="account_product_move_search_view">
+        <field name="name">account.product.move.search</field>
+        <field name="model">account.product.move</field>
+        <field name="arch" type="xml">
+            <search string="Search">
+                <field name="name" />
+                <field
+                    name="product_tmpl_id"
+                    string="Product Default Code"
+                    filter_domain="[('product_tmpl_id.default_code', 'ilike', self)]"
+                />
+                <field
+                    name="product_tmpl_id"
+                    string="Product Name"
+                    filter_domain="[('product_tmpl_id.name', 'ilike', self)]"
+                />
+            </search>
+        </field>
+    </record>
+    <!-- Action and menu -->
+    <record model="ir.actions.act_window" id="account_product_move_act_window">
+        <field name="name">Account Product Move</field>
+        <field name="res_model">account.product.move</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+    <menuitem
+        id="menu_product_move"
+        name="Extra Product Moves"
```

Comment:
I still feel it's a bit of a confusing name, but I can't think of anything better. Maybe Product-related Journal Entry templates.